tests/conftest.py

```diff
@@ -1730,8 +1730,17 @@ def __dut_reload(duts_data, node=None, results=None):
         logger.error('Missing kwarg "node" or "results"')
         return
     logger.info("dut reload called on {}".format(node.hostname))
-    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"], indent=4),
+    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"]["host"], indent=4),
               dest='/etc/sonic/config_db.json', verbose=False)
+
+    if node.is_multi_asic:
+        for asic_index in range(0, node.facts.get('num_asic')):
+            asic_ns = 'asic{}'.format(asic_index)
+            asic_cfg_file = "/tmp/{}_config_db{}.json".format(node.hostname, asic_index)
+            with open(asic_cfg_file, "w") as outfile:
+                outfile.write(json.dumps(duts_data[node.hostname]['pre_running_config'][asic_ns], indent=4))
+            node.copy(src=asic_cfg_file, dest='/etc/sonic/config_db{}.json'.format(asic_index), verbose=False)
+
```

Comment:
asic_cfg_file is to be removed?

Reply:
Added code to remove the file using os.remove